### PR TITLE
Changed a few things in your_first_game

### PR DESCRIPTION
**Pull request by TwistedTwigleg**
I've made a few changes to the 'your first game' tutorial:
* Changed the bit about saving the scene from `"...or press Meta-s"` to `"...or
press Control-s for Windows/Linux or Command-s on Mac"`
* Added a note about how `$` is the same as `get_node()`
* Added a couple warnings about the error `"node not found"`
They explain how the node name in `$` (or `get_node()`) is case sensitive and
encourages them to ensure the name in the script is the same as in the scene

All but the changes in the saving bit are based on [this forum post](https://godotdevelopers.org/forum/discussion/18906/docs-tutorial-problem-with-code) and [this issue](https://github.com/godotengine/godot-demo-projects/issues/72) on the demo repository.

On an aside, the tutorial is quite nice! Kudos to whoever wrote it! :+1:

### CODE REVIEW

File: learning/step_by_step/your_first_game.rst [hunk 1]
```diff
@@ -79,7 +79,7 @@ are not selectable."
 
 .. image:: img/lock_children.png
 
-Save the scene (click Scene -> Save, or press ``Meta-s``).
+Save the scene (click Scene -> Save, or press ``Control-s`` on Windows/Linux or ``Command-s`` on Mac).
```
Comment (cbscribe) [inline learning/step_by_step/your_first_game.rst]: Good call! The menu used to say `Meta-s`. I'm glad it's changed.

File: learning/step_by_step/your_first_game.rst [hunk 2]
```diff
@@ -218,13 +221,15 @@ We can prevent that if we *normalize* the velocity, which means we set
 its *length* to ``1``, and multiply by the desired speed. This means no
 more fast diagonal movement.
 
+
```
Comment (cbscribe) [inline learning/step_by_step/your_first_game.rst]: Remove this extra blank line.

File: learning/step_by_step/your_first_game.rst [hunk 3]
```diff
@@ -218,13 +221,15 @@ We can prevent that if we *normalize* the velocity, which means we set
...
 .. tip:: If you've never used vector math before (or just need a refresher)
          you can see an explanation of vector usage in Godot at :ref:`doc_vector_math`.
          It's good to know but won't be necessary for the rest of this tutorial.
 
 We also check whether the player is moving so we can start or stop the
 AnimatedSprite animation.
 
+
```
Comment (cbscribe) [inline learning/step_by_step/your_first_game.rst]: This one too

File: learning/step_by_step/your_first_game.rst [hunk 4]
```diff
@@ -240,6 +245,10 @@ and use ``clamp()`` to prevent it from leaving the screen:
 Click "Play the Edited Scene. (F6)" and confirm you can move the player
 around the screen in all directions.
 
+.. warning:: If you get any errors in the debugger saying "Node not found"
+             check the script to see if it has the `exact` same name as the node in the scene.
+             (``$`` and ``get_node()`` are `case sensitive!`)
+
```
Comment (cbscribe) [inline learning/step_by_step/your_first_game.rst]: I would reword this. When you use an invalid node name `get_node()` returns `null`. The error message you get will be "Invalid <something> (on base: 'null instance')"

If you get an error in the "Debugger" panel that refers to a "null instance", this likely means you spelled the node name wrong. Node names are case sensitive and ``$NodeName`` or ``get_node("NodeName")`` must match the name you see in the Scene tree.

File: learning/step_by_step/your_first_game.rst [hunk 5]
```diff
@@ -750,6 +759,10 @@ sync with the changing score:
 Now you're ready to play! Click the "Play the Project" button. You will
 be asked to select a main scene, so choose ``Main.tscn``.
 
+.. warning:: If you get any errors in the debugger saying "Node not found"
+             check the script to see if it has the `exact` same name as the node in the scene.
+             (``$`` and ``get_node()`` are `case sensitive!`)
+
```
Comment (cbscribe) [inline learning/step_by_step/your_first_game.rst]: I don't think it's necessary to repeat the same warning twice. See above.

File: learning/step_by_step/your_first_game.rst (hunk 4 above)
Comment (TwistedTwigleg) [reply learning/step_by_step/your_first_game.rst]: Sounds good! I can't fix this for a bit, but when I can I'll change it!

File: learning/step_by_step/your_first_game.rst (hunk 5 above)
Comment (TwistedTwigleg) [reply learning/step_by_step/your_first_game.rst]: Okay. Should I place the warning here or above?

Comment (cbscribe) [reply learning/step_by_step/your_first_game.rst]: Above makes the most sense. It's where they're likeliest to run into the issue.

File: learning/step_by_step/your_first_game.rst (hunk 1 above)
Comment (cbscribe) [reply learning/step_by_step/your_first_game.rst]: Another small thing I noticed: the menus use capitals and `+` signs now too: ``Command+S``

Comment (TwistedTwigleg) [reply learning/step_by_step/your_first_game.rst]: I'll change these to use the `+` sign.

File: learning/step_by_step/your_first_game.rst (hunk 5 above)
Comment (TwistedTwigleg) [reply learning/step_by_step/your_first_game.rst]: Okay. I'll remove this one then.

**Comment by TwistedTwigleg** [comment]
Okay, I made the changes!

I also moved the tip about `$` and `get_node` down a little so it's closer to where the code is explained, and tried to explain what it does. As Kidscancode pointed out in the forum post, it's not really a clarification if it isn't explained. Hopefully now it's a little more clear on what it does. I'm not 100% sure on the wording though.

File: learning/step_by_step/your_first_game.rst [hunk 6]
```diff
@@ -225,6 +225,12 @@ more fast diagonal movement.
 We also check whether the player is moving so we can start or stop the
 AnimatedSprite animation.
 
+.. tip:: ``$`` returns the node at the relative path from this node, or will return ``null`` if the node is not found.
+         Since AnimatedSprite is a child of this node, we can just use ``$AnimatedSprite``.
```
Comment (cbscribe) [inline learning/step_by_step/your_first_game.rst]: mixed tense: "returns" vs. "will return". Change the latter to "returns ``null``"

It may be unclear what "this node" refers to. I would say "the current node"

**Comment by TwistedTwigleg** [comment]
All fixed! Thanks cbscribe!

**Comment by cbscribe** [comment]
Thanks for your help!
